vehicle/mkvi/software/sensing/sensing.yml

```diff
@@ -0,0 +1,19 @@
+name: brakelight_bspd
+# subscribe:
+#   - name: air_control_critical
+#     mob: 1
+publish:
+  - name: sensing
+    id: 0xb 
```

Comment:
Make sure you update the CAN ID based on our address sheet (https://docs.google.com/spreadsheets/d/12KenzPC5moFQULhOhoBm-pcmC4ligmMhP_M-TPqN-CI/edit#gid=1599758060). Not sure how up to date the Sensing section is on the document based on this year's architecture.

Reply:
@jack-greenberg could I just assign myself on the spreadsheet for this?